ocs_ci/ocs/resources/packagemanifest.py

```diff
@@ -125,13 +127,21 @@ def get_current_csv(self, channel=None):
         Raises:
             ResourceNameNotSpecifiedException: in case the name is not
                 specified.
+            ChannelNotFound: in case the required channel doesn't exist.
 
         """
         self.check_name_is_specified()
         channel = channel if channel else self.get_default_channel()
-        for _channel in self.get_channels():
+        channels = self.get_channels()
+        for _channel in channels:
             if _channel['name'] == channel:
                 return _channel['currentCSV']
+        channel_names = [_channel['name'] for _channel in channels]
+        raise ChannelNotFound(
+            f"Channel: {channel} not found in available channels: "
+            f"{channel_names}"
+        )
+
```

Comment:
Tox is failing cause you have too many blank lines here

Reply:
I noticed it once I sent the PR, did change locally but forgot to push the change.  Anyway the PR will be blocked by the travis hook. For such issues I think no need to require change on the PR as this probably block the PR till you will remove this request for change but I am not sure if I can force merge if change required. Let's see otherwise I will need to wait for another X hours you will be online or open new one PR.